model_compression_toolkit/core/common/fusion/fusing_info.py

```diff
@@ -228,6 +228,22 @@ def is_node_in_fused_op(self, node: 'BaseNode') -> bool:
         """
         return any(node in nodes for nodes in self.fusing_data.values())
 
+    def is_quantized_node_in_fln(self, node: 'BaseNode') -> bool:
+        """
+        Check whether a node inside an FLN and should be quantized.
+
+        Args:
+            node (BaseNode): The node to check.
+
+        Returns:
+            bool: True if the node is in any fused operation and should be quantized.
+        """
+        if self.is_node_in_fused_op(node):
+            node_q_cfg = self.fused_op_id_to_quant_config[self.node_to_fused_node_map[node.name]]
+            return node_q_cfg is not None and node_q_cfg.enable_activation_quantization
```

Comment:
must we carry the op_cfg, and not put the data on the node? probably require another quant_mode: FLN_NOQUANT...
It will make the changes in the model_collector redundant